packages/cadl-ranch-specs/http/azure/core/lro/rpc-legacy/main.tsp

```diff
@@ -143,5 +143,5 @@ interface CreateResourcePollViaOperationLocation {
   @pollingOperation(CreateResourcePollViaOperationLocation.getJob)
   @doc("Creates a Job")
   @route("/jobs")
-  createJob is LongRunningRpcOperation<JobData, JobResult, JobResult>;
+  createJob is Foundations.Operation<JobData, AcceptedResponse & Foundations.LongRunningStatusLocation<JobResult>>;
```

Comment:
Template can't describe whether the initial response has body. Here is the reference: https://azure.github.io/typespec-azure/docs/howtos/Azure%20Core/long-running-operations#example-3---link-to-statusmonitor-in-the-location-header